src/storage.rs

```diff
@@ -798,6 +822,40 @@ impl Store {
 
         new_imports
     }
+
+    /// Called to ensure that there is publisher information in the store's live
+    /// imports for the given crate. This is used when adding new wildcard
+    /// audits from `certify`.
+    pub fn ensure_publisher_versions(
+        &mut self,
+        cfg: &Config,
+        network: Option<&Network>,
+        package: PackageStr<'_>,
+    ) -> Result<&[CratesPublisher], CertifyError> {
+        info!("ensure_publisher_versions");
```

Comment:
This would be more appropriate as `#[tracing::instrument]` on the function, if kept at all.

Reply:
Oops, thanks for finding my debugging logs which I forgot to remove :p